src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/StreamHelper.cs

```diff
@@ -225,7 +230,15 @@ private void Initialize()
                 {
                     if (_ownerCmdlet != null)
                     {
-                        record.StatusDescription = StringUtil.Format(WebCmdletStrings.ReadResponseProgressStatus, totalLength);
+                        record.StatusDescription = StringUtil.Format(
+                            WebCmdletStrings.ReadResponseProgressStatus,
+                            DisplayHumanReadableFileSize(totalLength),
+                            DisplayHumanReadableFileSize(_contentLength == null ? 0 : (long)_contentLength));
```

Comment:
If we don't know expected length we get "1 Mb of 0" - this looks bad.
I suggest something like "1 Mb of ???????" .

Also please use modern syntax "is null"/"is not null" everywhere.

Reply:
Happy to change it whatever if people are happy with just a `???` instead of the `0`.

Reply:
"1 Mb of 0" looks as a curiosity for me. :-) 
We could use "1 Mb of <unknown>" but ??????? looks more international :-) - no need to translate.

Reply:
Applied suggestion in latest comment